pkgs/games/commandergenius/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, SDL2_image, pkgconfig
+, libvorbis, libogg, mesa, boost, curl, zlib, cmake }:
+
+
+stdenv.mkDerivation {
+  name = "commandergenius194";
```

Comment:
The name and version should be separated by a dash for `nix-env -u` to work properly. You can also re-use the version variable you have below (thank to the `rec`). So that would look like: `name = "commandergenius-${version}";`